### ISSUE TITLE
async & await not work in ie8...

### ISSUE THREAD

**Issue by txiaocao** [Microsoft/TypeScript#13299]
**TypeScript Version:**  2.1.4 / nightly (2.2.0-dev.201xxxxx)

**Code**

```ts
class Test {
    public async init() {
        alert("pending")
        await this.a()
        alert("finish")
    }
    public async a() {
        return new Promise((resolve) => {
            setTimeout(function () {
                alert("a")
                resolve()
            }, 1000)
        })
    }
}

async function foo() {
    alert("foo")
    await new Test().init()
}

foo().then(undefined,() => {
    alert(Promise)
})

```

**Expected behavior:**
It can work in > IE9,but in IE8 nothing happend...not error,code do not run...
**Actual behavior:**

**Comment by DanielRosenwasser** [comment]
Are you using a polyfill for `Promise`s? TypeScript expects something like `core-js`, `es6-promise`, or something similar to be loaded at runtime.

**Comment by txiaocao** [comment]
sorry,i seem this is a duplicate issues #11280... 
it is fixed in typescript 2.1.5
thank you,your are best ^_^
